src/chains/filecoin/filecoin/src/schema.ts

```diff
@@ -1,18 +1,28 @@
 import FilecoinApi from "./api";
-import { Schema } from "@filecoin-shipyard/lotus-client-schema";
+import LotusSchema, { Schema } from "@filecoin-shipyard/lotus-client-schema";
 
 const GanacheSchema: Schema = {
   methods: {}
 } as Schema;
 
+const combinedMethods = {
+  ...LotusSchema.mainnet.fullNode.methods,
+  ...LotusSchema.mainnet.storageMiner.methods,
+  ...LotusSchema.mainnet.gatewayApi.methods,
+  ...LotusSchema.mainnet.walletApi.methods,
+  ...LotusSchema.mainnet.workerApi.methods
+};
+
 // Use the FilecoinAPI to create a schema object representing the functions supported.
 for (const methodName of Object.getOwnPropertyNames(FilecoinApi.prototype)) {
-  if (methodName == "constructor") {
+  if (!methodName.startsWith("Filecoin.")) {
```

Comment:
> It looks like this Schema won't include methods with the prefix Ganache, assuming we continue with that prefix (see earlier discussion).

That is correct. The way I interpreted your intended purpose of this object was to initialize `lotus-client-rpc`, which [prefixes](https://github.com/filecoin-shipyard/js-lotus-client-rpc/blob/master/index.js#L38) all methods with `Filecoin.`.

If we wanted `lotus-client-rpc` to actually be able to call the mine tipset function, we would need to change the prefix to `Filecoin.` despite it being part of the main namespace.

However, this function will not need to change regardless if we change the prefix if the intention is to use it with initializing `lotus-client-rpc`. This function just tells that library which `Filecoin.` methods are available, since it will only execute those methods due to the hardcoded prefix in their code.

Reply:
All good for leaving it how it is. Thanks @seesemichaelj, and sorry for the delay.